nubis/builder/provisioners.json

```diff
@@ -2,26 +2,29 @@
 "provisioners": [
   {
     "type": "shell",
-    "inline": ["sudo chmod 777 /var/www/dpaste"],
-    "order": "20"
+    "inline": [
+      "sudo mkdir -p /var/www/dpaste",
```

Comment:
The reason for this is one of ordering. Before, you were running the puppet first, then copying over the dpaste stuff, then running install.sh

Now, the order is reversed, as puppet can't run pip requirements.txt until the dpaste code is there. So the order needs to change to upload code then run puppet then run install.sh

It's going to get cleaner still once all is in puppet, but I am not done with that yet.